src/cpp/session/modules/SessionPackages.R

```diff
@@ -797,74 +815,31 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    .rs.setVar("ignoreNextLoadedPackageCheck", FALSE)
    if (ignore)
       return(FALSE)
-
-   # if the default set of namespaces in rstudio are loaded
-   # then skip the check
-   defaultNamespaces <- c("base", "datasets", "graphics", "grDevices",
-                          "methods", "stats", "tools", "utils")
-   if (identical(defaultNamespaces, loadedNamespaces()) &&
-       length(.dynLibs()) == 4)
-      return(FALSE)
```

Comment:
This check is seemingly not correct, minimally because `loadedNamespaces()` does not sort its return value:

```
$ R --vanilla -s -e 'loadedNamespaces()'
[1] "compiler"  "graphics"  "utils"     "grDevices" "stats"     "datasets"
[7] "methods"   "base"
```

and

```
$ R --vanilla -s -e '.dynLibs()'
                                                                                          Filename
1     /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/methods/libs/methods.so
2         /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/utils/libs/utils.so
3 /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/grDevices/libs/grDevices.so
4   /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/graphics/libs/graphics.so
5         /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/stats/libs/stats.so
  Dynamic.Lookup
1          FALSE
2          FALSE
3          FALSE
4          FALSE
5          FALSE
```